### PR TITLE
Add Katya

### PR DESCRIPTION
**Pull request by KatiaZzz**

### CODE REVIEW

**Comment by ava8katushka** [comment]
Please resolve the conflicts :)

File: README.md [hunk 1]
```diff
@@ -7,8 +7,8 @@
...
 | - |:-----:| --------------------------------------:|
 | 0 | Ава   | Три идиота                             |
-| 1 | Павел | Криминальное чтиво                     |
-| 2 |  Камила      |       Кролик ДжоДжо             |
-| 3 | Андрей| Храброе Сердце                         |
- 
-
+| 1 | Катя  | Такси                                  |                          
+| 2 | Павел | Криминальное чтиво                     |                                 
+| 3 | Павел | Криминальное чтиво                     |
```
Comment (ava8katushka) [inline README.md]: why pavel is here twice?

**Review by ava8katushka — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (KatiaZzz) [reply README.md]: I guess he added himself there twice

Comment (ava8katushka) [reply README.md]: the green text is your changes. Please, be accurate and resolve.

File: README.md [hunk 2]
```diff
@@ -9,6 +9,5 @@
 | 0 | Ава   | Три идиота                             |
 | 1 | Катя  | Такси                                  |
```
Comment (ava8katushka) [inline README.md]: Almost there! But you are not the first. Please add yourself to the end of the list :)

**Pull request merged**